app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name, health=100,):
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self):
+        return f"{{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}}"
+
+
+class Herbivore(Animal):
+    hidden = True
```

Comment:
The `hidden` attribute should be an instance attribute, not a class attribute. This will ensure that each `Herbivore` instance can have its own `hidden` state.